polyglot/piranha/test-resources/java/feature_flag_system_1/control/expected/XPFlagCleanerPositiveCases.java

```diff
@@ -68,10 +68,14 @@ public void conditional_with_else_contains_stale_flag() {
     System.out.println("Hi world");
   }
 
-  public void complex_conditional_contains_stale_flag(boolean tBool) {
+    public void complex_conditional_contains_stale_flag(boolean tBool) {
```

Comment:
Formatting seems off for this example. There is an extra space before `public`, and the if is not indented. Not exactly a major issue, but is there any reason this can't be more standardly formatted? 🙂

p.s. Same in other places below. Yet I checked these files and sampling a bit at random, the rest of the Java code looks properly indented.